привет

Soburov opened the pull request:
поделись плиз.

VICLER committed:
add new OLED beta

VICLER committed:
upd 2.4

- edit timoutOff
- add option to enable/disable rainbow flow light
- small bugfixes

VICLER committed:
upd 1.1

VICLER committed:
upd 2.4

VICLER committed:
upd 2.0

VICLER committed:
Update ServoSmooth.cpp

- disabled acceleration

VICLER committed:
add SSD1306Ascii

VICLER committed:
Update c_func.ino

VICLER committed:
Update d_func.ino

VICLER committed:
upd v2.5

- KEEP_POWER rework

VICLER committed:
upd v2.5

VICLER committed:
upd v2.5

VICLER committed:
upd v2.5

VICLER committed:
Update ServoSmooth.cpp

VICLER committed:
Update c_func.ino

VICLER committed:
upd v2.6

- ability to reset settings with a button
- bug fixes

VICLER committed:
Update GyverDrink_VICLER_MOD.ino

VICLER committed:
Update c_func.ino

VICLER committed:
upd v1.2

VICLER committed:
upd v1.2

VICLER committed:
upd v1.3

add keep_power

VICLER committed:
upd v1.3

VICLER committed:
upd v1.3

VICLER committed:
upd v1.3

VICLER committed:
upd v1.3

VICLER committed:
upd SSD1306Ascii

added font

VICLER committed:
upd SSD1306Ascii

VICLER committed:
upd 1.4

add display inversion
bugfixes and improvements

VICLER committed:
upd 1.4

VICLER committed:
upd 1.4

VICLER committed:
upd 1.4

VICLER committed:
upd StepMot

VICLER committed:
upd ServoSmooth

VICLER committed:
upd v1.5

VICLER committed:
upd v2.1

VICLER committed:
upd v2.7

add battery monitoring
service mode reworked

VICLER committed:
Update d_func.ino

VICLER committed:
Update GyverDrink_VICLER_MOD.ino

VICLER committed:
Update c_setup.ino

VICLER committed:
Update GyverDrink_VICLER_MOD_OLED.ino

VICLER committed:
Update README.md

VICLER committed:
Update README.md

VICLER committed:
upd 1.6

servo calibration rework
added battery calibration option on start

VICLER committed:
upd v2.7.1

VICLER committed:
upd v2.7.1

VICLER committed:
upd v1.6.1

VICLER committed:
upd v1.7

bugfixes

VICLER committed:
upd v 1.7

VICLER committed:
upd v2.8

VICLER committed:
Update ServoSmooth.cpp

VICLER committed:
upd v1.7

VICLER committed:
upd v2.8

VICLER committed:
upd ServoSmooth

VICLER committed:
upd v1.7

VICLER committed:
upd v1.7

VICLER committed:
upd v1.8

VICLER committed:
upd ServoSmooth

VICLER committed:
upd v1.9

improved servo movements in service mod for slow servos
bugfixes

VICLER committed:
upd v2.9

fixed servo movements in service mode for slow servos
bug fixes

VICLER committed:
upd v3.0

VICLER committed:
upd 3.0.1

stability improvement

VICLER committed:
upd v2.0

bugfixes and stability improvement

VICLER committed:
upd v3.0.1

stability improvement

VICLER committed:
Update GyverDrink_VICLER_MOD_OLED.ino

cosmetic changes

VICLER committed:
upd v2.1

- set servo speed from menu
- bugfixes

VICLER committed:
del

VICLER committed:
upd v3.1

change animation on every start

VICLER committed:
upd v3.1.1

VICLER committed:
upd

clean code from warnings

VICLER committed:
upd v2.1.1

VICLER committed:
del

VICLER committed:
upd

VICLER committed:
upd

VICLER committed:
upd

VICLER committed:
upd

VICLER committed:
Update README.md

VICLER committed:
Update README.md

VICLER committed:
Update README.md

VICLER committed:
Update Config.h

VICLER committed:
Update GyverDrink_VICLER_MOD.ino

VICLER committed:
upd 2.2

VICLER committed:
upd v3.2

- fix resetting parameters on startup
- saving work mode to eeprom

VICLER committed:
upd v2.2

- fix reseting parameters on startup
- saving last work mode to eeprom

VICLER committed:
upd v2.3

VICLER committed:
upd v3.2.1

VICLER committed:
upd v 2.3.1

VICLER committed:
upd v3.2.1

VICLER committed:
upd v2.3.1

VICLER committed:
Update .gitignore

VICLER committed:
upd

VICLER committed:
Update tm1637.ino

VICLER committed:
upd

VICLER committed:
add

VICLER committed:
Update c_func.ino

VICLER committed:
upd

VICLER committed:
upd

VICLER committed:
Create LICENSE

VICLER committed:
upd

VICLER committed:
del

VICLER committed:
add v1.0

VICLER committed:
upd v4.0

VICLER committed:
upd v4.0

VICLER committed:
upd v4.0

VICLER committed:
Update Config.h

VICLER committed:
upd v4.1

VICLER committed:
upd v41

VICLER committed:
upd v4.1

VICLER committed:
Update GyverDrink_VICLER_MOD.ino

VICLER committed:
upd 4.2

VICLER committed:
upd v4.3

VICLER committed:
Update e_control.ino

VICLER committed:
upd v4.4

VICLER committed:
upd v2.2

VICLER committed:
upd v4.5

VICLER committed:
upd v4.5.1

VICLER committed:
upd v4.5.1

VICLER committed:
upd v4.5.1

VICLER committed:
add stl

VICLER committed:
upd v4.6

VICLER committed:
upd v4.6

VICLER committed:
upd v4.7

VICLER committed:
upd v4.7

VICLER committed:
upd v4.8

VICLER committed:
upd v4.9

VICLER committed:
upd v 4.9

VICLER committed:
upd v 4.9

VICLER committed:
upd v4.9

VICLER committed:
Update README.md

VICLER committed:
Update GyverDrink_VICLER_MOD.ino

VICLER committed:
Update Config.h

VICLER committed:
upd v5.0

VICLER committed:
upd v5.1

VICLER committed:
upd v5.1

VICLER committed:
upd v5.2

VICLER committed:
upd v5.2

VICLER committed:
Create ContrastTestWire.ino

VICLER committed:
upd v5.3

VICLER committed:
Update d_func.ino

VICLER committed:
Create SSD1306_Cover.stl

VICLER committed:
Create SSD1306_Cover_v2.stl

VICLER committed:
upd v5.5

VICLER committed:
upd v5.6

VICLER committed:
upd v5.6

VICLER committed:
upd v5.7

VICLER committed:
upd 5.7

VICLER committed:
upd v5.7

VICLER committed:
upd v5.7

VICLER committed:
upd readme

ange007 committed:
Обновление Readme для того кто впервые видит репу

Мне кажется так будет несколько понятней что к чему. А то сам как первый раз пришёл в данную репу - не мог понять что к чему.

VICLER committed:
upd

VICLER committed:
upd

VICLER committed:
Update README.md

VICLER committed:
Update README.md

VICLER committed:
upd

VICLER committed:
Merge branch 'master' into patch-1

VICLER committed:
Merge pull request #1 from ange007/patch-1

Обновление Readme для того кто впервые видит репу

VICLER committed:
upd

clean

VICLER committed:
upd

VICLER committed:
Update README.md

VICLER committed:
upd v5.8

VICLER committed:
upd v5.8

VICLER committed:
Update README.md

VICLER committed:
upd v5.9

VICLER committed:
upd v6.0

VICLER committed:
upd v6.1

VICLER committed:
upd v6.1

VICLER committed:
upd v6.1.1

VICLER committed:
upd v6.2

VICLER committed:
upd

VICLER committed:
Update README.md

VICLER committed:
Update .gitignore

VICLER committed:
upd

VICLER committed:
Update scheme2.png

VICLER committed:
Update README.md

VICLER committed:
upd

VICLER committed:
upd

VICLER committed:
upd

VICLER committed:
upd

VICLER committed:
Update README.md

VICLER committed:
upd

VICLER committed:
upd v6.3

VICLER committed:
upd 6.31

VICLER committed:
upd

VICLER committed:
upd

VICLER committed:
Update README.md

VICLER committed:
upd

VICLER committed:
upd v6.32

VICLER committed:
upd v6.4

VICLER committed:
upd v6.4

VICLER committed:
upd v6.41

VICLER committed:
upd v6.5

VICLER committed:
upd v6.51

VICLER committed:
Update Config.h

VICLER committed:
upd v6.6

VICLER committed:
upd v6.6

VICLER committed:
upd v6.6

VICLER committed:
upd v6.6

VICLER committed:
upd v6.7

VICLER committed:
upd v6.8

VICLER committed:
upd v6.9

VICLER committed:
upd v 6.9.1

VICLER committed:
upd v6.9

11.11.2020

VICLER committed:
upd v6.9

07.12.2020

AlexGyver commented:
Батюшки, может просто сослаться на проект виклера?)

VICLER committed:
upd v7.0

+ DISPLAY_SESSION_VOLUME
+ button debounce time 100ms

VICLER commented:
да уж, может показаться, что я без перерывов на обед пишу этот проект) соединять всё же не стоит

AlexGyver commented:
> да уж, может показаться, что я без перерывов на обед пишу этот проект) соединять всё же не стоит

У тебя на какой стадии? Можно полноценно ссылаться?)

VICLER committed:
upd v7.1

changed display selecting method
fix SERVO_AUTO_POWER for using with power driver

VICLER commented:
> > да уж, может показаться, что я без перерывов на обед пишу этот проект) соединять всё же не стоит
> 
> У тебя на какой стадии? Можно полноценно ссылаться?)

в постоянной разработке (сейчас вроде довольно стабильно всё). На данный момент имеется две отдельные прошивки под разные приводы (серво и шаговик). Законченно будет когда добавлю шаговик в основную прошивку, главное чтобы места хватило.

что именно имеешь ввиду под ссылаться? Откуда? Ты вроде уже ссылался на этот проект на сайте и тут

VICLER committed:
upd v7.1

- fix volume calibration screen

VICLER committed:
Update a_display.ino

AlexGyver commented:
> > > да уж, может показаться, что я без перерывов на обед пишу этот проект) соединять всё же не стоит
> > 
> > 
> > У тебя на какой стадии? Можно полноценно ссылаться?)
> 
> в постоянной разработке (сейчас вроде довольно стабильно всё). На данный момент имеется две отдельные прошивки под разные приводы (серво и шаговик). Законченно будет когда добавлю шаговик в основную прошивку, главное чтобы места хватило.
> 
> что именно имеешь ввиду под ссылаться? Откуда? Ты вроде уже ссылался на этот проект на сайте и тут

А, уже сослался) Всего не упомнить

VICLER committed:
upd schemes

VICLER committed:
fix warnings

VICLER committed:
Update d_func.ino

VICLER committed:
upd v2.3

fix ignoring endstop while homing

VICLER committed:
upd 7.2

fixed pause by activating a shot sensor while already moving servo

VICLER committed:
upd 7.2

add display version number on startup

VICLER committed:
upd 7.3

added: Stepper

VICLER committed:
Update Config.h

VICLER committed:
Update GyverDrink_VICLER_MOD.ino

VICLER committed:
upd v7.3

added smooth stepper algorithm option

VICLER committed:
upd v7.31

added servo min, max pulse range configuration

VICLER committed:
7.31

VICLER committed:
7.4

optimization

VICLER committed:
Merge pull request #2 from VICLER/dev

upd v7.4

VICLER committed:
upd v7.5

fixed volume counter
faster displaying big numbers >99

VICLER committed:
upd 7.5

fix agle display in service settings

VICLER committed:
upd 7.5 beta 3

+ display anti-burn in standby mode

VICLER committed:
Update a_display.ino

VICLER committed:
Merge pull request #4 from VICLER/dev

upd v7.5

VICLER committed:
upd 7.5

add reset eeprom while holding the main button for 5s

VICLER committed:
Merge pull request #5 from VICLER/dev

upd v7.5

VICLER committed:
7.6

fixed display float numbers > 9

VICLER committed:
upd 7.5.1

changed spi display connection

VICLER committed:
upd v7.6 beta1

added stepper endstop homing
fixed prepump mode
display speed improvements

VICLER committed:
Update d_func.ino

VICLER committed:
upd 7.6 beta2

changed time for double click to toggle work mode

VICLER committed:
upd 7.6 beta3

fix prepump counter

VICLER committed:
Merge pull request #6 from VICLER/dev

upd 7.6

VICLER committed:
Update .gitignore

VICLER committed:
Update .gitignore

VICLER committed:
Update .gitignore

VICLER committed:
upd v7.7

- memory optimization
- fixed fast mode switch
- fixed microWire library for using with Arduino IDE 1.8.15

VICLER committed:
Merge pull request #7 from VICLER/dev

upd 7.7

VICLER committed:
upd v7.8

added Servo accelerations. Parameter MOTOR_ACCEL
added parameter PAUSE_TIME. This is a time delay before moving to the next shot
added parameter PREPUMP_VOLUME. This allows to do a prepump automaticaly

VICLER committed:
upd 7.8.1

fixed english menu page

VICLER committed:
upd 7.8 beta2

fixed compile error with selected TM1637 display

VICLER committed:
upd

deleted BIgIcon36x40

VICLER committed:
Create BigIcon36x40.h

VICLER committed:
Update v 7.8

- added Servo accelerations. Parameter MOTOR_ACCEL
- added parameter PAUSE_TIME. This is a time delay before moving to the next shot
- added parameter PREPUMP_VOLUME. This allows to do a prepump automaticaly
- fixed english menu page
- fixed compile error with selected TM1637 display
- fixed naming of BigIcon36x40.h

VICLER committed:
Update .gitignore

VICLER committed:
Update .gitignore

VICLER committed:
delete stepMot

VICLER committed:
Update .gitignore
